src/submission/forms.py

```diff
@@ -33,6 +33,11 @@ def __init__(self, *args, **kwargs):
         self.fields['competing_interests'].label = ''
         self.fields['comments_editor'].label = ''
 
+        try:
+            journal.submissionconfiguration
+        except models.SubmissionConfiguration.DoesNotExist:
+            models.SubmissionConfiguration.objects.create(journal=journal)
+
```

Comment:
When do we expect this to happen?

Reply:
It happened to me yesterday. Basically when this model was introduced a Signal should have been added to generate them automatically but it wasn't so there may be journals out there without a configuration.

Reply:
But this code will live here forever, shouldn't we have it on some sort of upgrade script/migration instead?